src/irc/core/irc-servers-setup.c

```diff
@@ -30,6 +30,29 @@
 #include "irc-servers.h"
 #include "sasl.h"
 
+#include <errno.h>
+
+static void read_password_file(char *str, char **password)
+{
+	char **values = g_strsplit(str, ":", -1);
+	char *path;
+	GError *err = NULL;
+
+	path = g_strdup(values[1]);
```

Comment:
I get this when testing with valgrind
==31466== 10 bytes in 1 blocks are definitely lost in loss record 1,699 of 12,860
==31466==    at 0x4C2DB8F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==31466==    by 0x59EF728: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==31466==    by 0x5A084FE: g_strdup (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==31466==    by 0x480743: read_password_file (irc-servers-setup.c:41)
==31466==    by 0x480743: sig_server_setup_fill_chatnet (irc-servers-setup.c:126)
==31466==    by 0x4ACA41: signal_emit_real (signals.c:242)
==31466==    by 0x4ACF9A: signal_emit (signals.c:286)
==31466==    by 0x4A8F5B: server_setup_fill_chatnet (servers-setup.c:213)
==31466==    by 0x4A8F5B: create_addr_conn (servers-setup.c:254)
==31466==    by 0x4AFD1D: get_server_connect (chat-commands.c:74)
==31466==    by 0x4AFE7F: cmd_connect (chat-commands.c:162)
==31466==    by 0x4ACA41: signal_emit_real (signals.c:242)
==31466==    by 0x4ACF9A: signal_emit (signals.c:286)
==31466==    by 0x497FF8: parse_command (commands.c:906)
==31466==    by 0x497FF8: event_command (commands.c:952)

not sure what it is complaining about